src/main/java/com/thg/accelerator23/connectn/ai/ajtracey/analysis/BoardLine.java

```diff
@@ -20,6 +20,10 @@ public BoardLine(Board board, Position currentPosition,
         this.movementFunction = movementFunction;
     }
 
+    public Position getCurrentPosition() {
```

Comment:
Wow, u totally copied me
<img width="538" alt="image" src="https://user-images.githubusercontent.com/114484105/211591723-49d82a95-fe59-4e70-8bf4-5e1766442334.png">

Reply:
looool, great minds, i din't even use it in the end XD